inc/functions.php

```diff
@@ -5033,6 +5017,43 @@ function build_theme_select($name, $selected="", $tid=0, $depth="", $usergroup_o
 }
 
 /**
+ * Get the theme data of a theme id.
+ *
+ * @param int $tid The theme id of the theme.
+ * @return boolean False if no valid theme, array With the theme data otherwise
```

Comment:
Should be `@return boolean|array False if no valid theme, Array with the theme data otherwise`